tests/unit/src/commands/decryptDeployKey.ts

```diff
@@ -57,6 +55,7 @@ registerSuite({
 		decryptDataObj.pipe.reset();
 	},
 
+	tests: {
 	'decryptDeployKey': (() => {
```

Comment:
Shouldn't this be indented?

Reply:
Yes. As soon as I address all the other feedback and get the go-ahead, I'll indent these blocks. Indenting them from the start makes it a nightmare to review the changes.